archetype-common/src/test/java/org/apache/maven/archetype/old/ArchetypeTest.java

```diff
@@ -381,15 +413,19 @@ public void testAddModuleToParentWithProfiles()
                 + "      </modules>\n"
                 + "    </profile>\n"
                 + "  </profiles>\n" +
-                PROJECT_ELEMENT_END;
+                "</project>";
 
-        StringWriter out = new StringWriter();
+        out = new StringWriter();
```

Comment:
you don't need to initialize this in the test

Reply:
You are absolutly right. Fixed all occurances.